### ISSUE TITLE
Class.dlrs.RollupService.Updater.updateRecords: line 1248, column 1

### ISSUE THREAD

**Issue by vhanson222** [afawcett/declarative-lookup-rollup-summaries#837]
We're receiving the error below when attempting to change a lookup field's value on some records.  Any idea what would cause this?

Error:Apex trigger dlrs_SBQQ_SubscriptionTrigger caused an unexpected exception, contact your administrator: dlrs_SBQQ_SubscriptionTrigger: execution of AfterUpdate caused by: System.DmlException: Update failed. First exception on row 1 with id aCsA00000001jpBKAQ; first error: FIELD_CUSTOM_VALIDATION_EXCEPTION, Attempt to de-reference a null object: []: Class.dlrs.RollupService.Updater.updateRecords: line 1248, column 1

**Comment by JimBTek** [comment]
Is it running as the user and the user does not have FLS for one of the fields in the rollup?

**Comment by vhanson222** [comment]
@JimBTek thanks for the response.  This occurred during a data update by one of our Users with System Administrator profile, so not a FLS issue.

We had to temporarily disable our DLRS rules to update the record.  Once the field was updated, we manually executed our DLRS rules for that record and no error occurred.  Seems like it has something to do with switching the value of that one lookup field.
